src/openrct2/core/Console.cpp

```diff
@@ -19,6 +19,7 @@ extern "C"
     #include "../platform/platform.h"
 }
 
+#include <android/log.h>
```

Comment:
What should we do with the console? Android doesn't have stdout/stderr. Just discard the changes and not have it in this PR?